src/DynamoWebServer/Messages/MessageHandler.cs

```diff
@@ -465,6 +473,44 @@ private string GetValue(NodeModel node)
             return data;
         }
 
+        private object GenerateWatchData(MirrorData data)
```

Comment:
Should this method returns `string` instead? Also the `var list` can be made `List<string>`, too, no?